security/README.md

```diff
@@ -27,7 +27,7 @@ It generates the private key and CSR (certificate signing request) locally, send
 Istio Security uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) to identify who runs the service:
 
 *   A service account in Istio has the format "spiffe://\<_domain_\>/ns/\<_namespace_>/sa/\<_serviceaccount_\>".
-    *   _domain_ is currently _cluster.local_. We will support customization of domain in the near future.
+    *   _domain_ is customizable via command line, defaulting to _cluster.local_.
```

Comment:
Do you want to add a link to where this command line flag is defined?

Reply:
@incfly do you mean link to Go code, a mention of the CLI parameter name, or do we have a description of all CLI flags somewhere (e.g., on Istio docs)?

Reply:
will add the CLI parameter name in the README